app/src/main/AndroidManifest.xml

```diff
@@ -10,7 +10,8 @@
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
-        android:theme="@style/AppTheme">
+        android:theme="@style/AppTheme"
+        android:usesCleartextTraffic="true">
```

Comment:
I would suggest to specify a domain here, otherwise you are allowing cleartext traffic for every domain which is not very secure, hence forbidden by default.

Reply:
I’ll go along with you on this one.